package.json

```diff
@@ -61,7 +62,7 @@
     "@types/papaparse": "^5.3.2",
     "@types/pluralize": "^0.0.29",
     "@types/react": "^16.9.49",
-    "@types/react-dom": "^16.9.8",
+    "@types/react-dom": "^18.2.4",
```

Comment:
I'm generally curious how to manage when a bunch of dependencies are updated. Are there general tests to run? Are there standard ways to figure out if things will clash in unexpected ways? *

Reply:
Normally we should do it a bit more consistently as we go along. We've left this way too late. I'd like to update more of the dependencies, but some of them will be quite a bit of work. React Router, for example, is significantly different now, and we'll probably have to re-write our custom "useRouter" hook, which will affect a lot of areas.